src/plot_api/plot_api.js

```diff
@@ -835,6 +835,10 @@ Plotly.redraw = function(gd) {
  */
 Plotly.newPlot = function(gd, data, layout, config) {
     gd = getGraphDiv(gd);
+
+    // remove gl contexts
+    Plots.cleanPlot([], {}, gd._fullData || {}, gd._fullLayout || {});
```

Comment:
Great!

@monfera would you adding a test case for checking that `Plotly.newPlot` does clear existing gl contexts similar to the one for [`Plots.cleanPlot`](https://github.com/plotly/plotly.js/blob/46bbbaebfd70ed28196df9c4993a1ded5292bebf/test/jasmine/tests/gl_plot_interact_test.js#L510) ? 

Thanks in advance.